packages/query/src/stacks/balance/account-balance.hooks.ts

```diff
@@ -43,6 +43,81 @@ export function createStxCryptoAssetBalance(
   };
 }
 
+export function useStxBalancesQueries(addresses: string[]) {
+  const client = useStacksClient();
+  const network = useCurrentNetworkState();
+
+  // to get these correctly more refactoring and combining of queries is needed
+  // this time in useMempoolTxsInboundBalance + useMempoolTxsOutboundBalance
+  const inboundBalance = createMoney(0, 'STX');
```

Comment:
To implement this properly it seems like I also will need to refactor `useMempoolTxsInboundBalance` and `useMempoolTxsOutboundBalance` in a similar way so that they can run `useQueries` and return combined totals if needed